webpack.config.js

```diff
@@ -63,6 +63,7 @@ const plugins = [
   new webpack.EnvironmentPlugin({
     NODE_ENV: 'development',
     FORCE_SCRIPT_NAME: '',
+    CTS_API_ENDPOINT: 'https://scaife-cts-dev.perseus.org/api/cts',
```

Comment:
@mjhea0: Thanks for tackling this; I intended to get back here but hadn't yet.  I change the default to `https://scaife-cts-dev.perseus.org/api/cts` (since that is what we have in [settings.py](https://github.com/scaife-viewer/scaife-viewer/blob/8b56dde2fa5f99b1a58cad6b986ce0cb0f83b910/scaife_viewer/settings.py#L277)

@jtauber: With webpack, we need to explicitly define environment variables within `EnvironmentPlugin` that we want to make available.